### DISCUSSION TITLE
How about Windows 10's taskbar progress bar?

### DISCUSSION THREAD

**Discussion by CptPakundo** [valinet/ExplorerPatcher#1606] [Ideas]
**Edit: Disregard, One forgotten 3rd party plugin for MusicBee made me mistakenly assume this feature doesn't properly work, when it actually does.**

Original post as follows:
One useful feature that doesn't seem to work with ExplorerPatcher that does exist in both Windows 10 and even Windows 11 is the progress bar you get on a program's icon in the taskbar, like when you're copying files or when you're hearing playing a media file in a media player.
Is there any way to restore that type of functionality while using ExplorerPatcher's Windows 10 style taskbar?
It's pretty much the one missing thing for my use cases.
![z0u2o](https://github.com/valinet/ExplorerPatcher/assets/32355614/633ebbe1-411b-47bf-b2db-5d70f4264181)

**Comment by CptPakundo** [comment]
Huh, I'm kind of surprised to see this didn't resonate with anyone else so far.

**Comment by mbartlett21** [comment]
@CptPakundo
It works fine for me:
![image](https://github.com/valinet/ExplorerPatcher/assets/29034492/bda3c037-ab87-40d0-9b14-013540a83645)
(Windows 11 22H2 build 22621.1992)

**Reply by CptPakundo** [reply]
Interesting - MPC-HC's progress bar also works for me, but MusicBee's does not.
I wonder what's the difference...

**Reply by pyrates999** [reply]
some applications have chosen not to implement it?

**Reply by CptPakundo** [reply]
Well, this is kind of embarrasing...
Apparently I just forgot to reinstall a plugin that actually added that in.
I never thought I'd make that kind of mistake, but here we are.
